sim/team-validator.ts

```diff
@@ -684,18 +684,15 @@ export class TeamValidator {
 				} else {
 					problems.push(`${species.name} is only obtainable from events - it needs to match one of its events, such as:`);
```

Comment:
```suggestion
					problems.push(`${species.name} is only obtainable from events - it needs to match one of its events:`);
```

"such as" no longer makes sense if you're listing all of the events.